util/chunk/chunk.go

```diff
@@ -25,8 +25,7 @@ import (
 
 var msgErrSelNotNil = "The selection vector of Chunk is not nil. Please file a bug to the TiDB Team"
 
-// Chunk stores multiple rows of data in Apache Arrow format.
-// See https://arrow.apache.org/docs/memory_layout.html
+// Chunk stores multiple rows of data.
```

Comment:
no need to remove the link. Actually, the internal memory layout is the same with apache arrow.